src/state/slices/portfolioSlice/portfolioSlice.ts

```diff
@@ -23,6 +23,10 @@ export const portfolio = createSlice({
       moduleLogger.info('clearing portfolio')
       return initialState
     },
+    setAccountMetadata: (state, { payload }: { payload: AccountMetadataById }) => {
```

Comment:
putting this here in the `portfolioSlice` over the `accountSpecifiers` slice as we want to kill `accountSpecifiers` slice as it's redundant and confusing